AnnService/inc/Core/SPANN/ExtraFullGraphSearcher.h

```diff
@@ -106,11 +106,18 @@ namespace SPTAG
                     auto curIndexFile = f_createAsyncIO();
                     if (curIndexFile == nullptr || !curIndexFile->Initialize(curFile.c_str(), std::ios::binary | std::ios::in, 
 #ifndef _MSC_VER
+#ifdef BATCH_READ
```

Comment:
This is the change that Qi told me to use to fix the Linux error I was getting with AsyncFileIOs.  I just commented out the previous code and added what Qi gave me as the fix.  Should I remove the commented code?

Reply:
It may be mostly some other files, they look to me like they are from other PRs/not related to this change, it might make the history hard to understand

`VectorSet.h`
`MemoryReader.h`
`Training.h`
`SSDIndex.h`
`DistanceUtils.cpp`
`SPANNIndex.cpp`
`VectorSet.cpp`
`TxtReader.cpp`
`XvecReader.cpp`
`Quantizer/main.cpp`

Reply:
I see, would it be better for me to check out a fresh version and replace the changed files again to avoid these confusing commits and rebasing issues?